package.json

```diff
@@ -26,7 +26,10 @@
     "twilio": "^2.11.1",
     "vorpal": "^1.12.0",
     "ware": "^1.3.0",
-    "ws": "^2.2.2"
+    "ws": "^2.2.2",
+	"node-ews": "^3.2.2",
+    "node-xmpp-caps": "0.0.2",
+    "simple-xmpp": "^1.3.0"
```

Comment:
Please put new dependencies to alphabetically correct positions.
Putting then at the bottom also creates an unnecessary conflict with #1154.